libs/blocks/modal/modal.js

```diff
@@ -137,7 +158,14 @@ export async function getModal(details, custom) {
     [...document.querySelectorAll('header, main, footer')]
       .forEach((element) => element.setAttribute('aria-disabled', 'true'));
   }
-
+  if (dialog.classList.contains('commerce-frame')) {
+    if (isInitialPageLoad) {
```

Comment:
The `isInitialPageLoad` does not do anything - as the event listener for the message is only set up once anyway.

Reply:
No,Without that i see that message event listeners are added multiple times.
<img width="1512" alt="Screenshot 2023-07-31 at 5 15 44 PM" src="https://github.com/adobecom/milo/assets/138484653/203cfd2f-75ec-4439-bb52-48b09653df7c">